### ISSUE TITLE
Static paths added to hyprpm Makefile

### ISSUE THREAD

**Issue by sjcobb2022** [hyprwm/Hyprland#7301]
### Regression?

No

### System Info and Version

<details> <summary>System/Version info</summary>

```sh
Hyprland, built from branch  at commit 01ff5fdf6a00a9231bd3b56400d8bcab378c4257  ().
Date: 2024-08-10
Tag: , commits: 01ff5fdf6a00a9231bd3b56400d8bcab378c4257

flags: (if any)


System Information:
System name: Linux
Node name: slaptop
Release: 6.9.12
Version: #1-NixOS SMP PREEMPT_DYNAMIC Sat Jul 27 09:36:19 UTC 2024


GPU information:
01:00.0 VGA compatible controller [0300]: NVIDIA Corporation GA106M [GeForce RTX 3060 Mobile / Max-Q] [10de:2520] (rev a1) (prog-if 00 [VGA controller])
06:00.0 VGA compatible controller [0300]: Advanced Micro Devices, Inc. [AMD/ATI] Cezanne [Radeon Vega Series / Radeon Vega Mobile Series] [1002:1638] (rev c5) (prog-if 00 [VGA controller])
NVRM version: NVIDIA UNIX x86_64 Kernel Module  550.78  Sun Apr 14 06:35:45 UTC 2024


os-release: ANSI_COLOR="1;34"
BUG_REPORT_URL="https://github.com/NixOS/nixpkgs/issues"
BUILD_ID="24.05.20240810.a781ff3"
DOCUMENTATION_URL="https://nixos.org/learn.html"
HOME_URL="https://nixos.org/"
ID=nixos
IMAGE_ID=""
IMAGE_VERSION=""
LOGO="nix-snowflake"
NAME=NixOS
PRETTY_NAME="NixOS 24.05 (Uakari)"
SUPPORT_END="2024-12-31"
SUPPORT_URL="https://nixos.org/community.html"
VERSION="24.05 (Uakari)"
VERSION_CODENAME=uakari
VERSION_ID="24.05"


plugins:

======Config-Start======


Config File: /home/sjcobb/.config/hypr/hyprland.conf: Read Succeeded
exec-once = /nix/store/8piinp0n24cl145qndjdfzja8x9s5x0k-dbus-1.14.10/bin/dbus-update-activation-environment --systemd DISPLAY HYPRLAND_INSTANCE_SIGNATURE WAYLAND_DISPLAY XDG_CURRENT_DESKTOP && systemctl --user stop hyprland-session.target && systemctl --user start hyprland-session.target
$mod=SUPER
animations {
  bezier=myBezier, 0.05, 0.9, 0.1, 1.05
  animation=windows, 1, 7, myBezier
  animation=windowsOut, 1, 7, default, popin 80%
  animation=windowsMove, 1, 7, myBezier
  animation=border, 1, 10, default
  animation=borderangle, 1, 8, default
  animation=fade, 1, 7, default
  animation=workspaces, 1, 6, default
  enabled=true
}

debug {
  disable_logs=false
}

decoration {
  drop_shadow=false
}

dwindle {
  preserve_split=true
  pseudotile=true
}

general {
  border_size=2
  gaps_in=5
  gaps_out=10
  layout=dwindle
}

gestures {
  workspace_swipe=true
  workspace_swipe_fingers=4
}

input {
  touchpad {
    disable_while_typing=false
    natural_scroll=true
  }
  follow_mouse=1
  kb_layout=us
  kb_model=
  kb_options=
  kb_rules=
  kb_variant=
  sensitivity=0
}

master {
}

misc {
  disable_hyprland_logo=true
  key_press_enables_dpms=true
  mouse_move_enables_dpms=true
  new_window_takes_over_fullscreen=2
  vrr=1
}


xwayland {
  force_zero_scaling=true
}
bind=$mod,M,exit,
bind=$mod,Return,exec,/nix/store/sgqh9ylizbyys6pc1f21z5ql3cy1bg3h-alacritty-0.13.2/bin/alacritty
bind=$mod,Q,killactive,
bind=$mod SHIFT,W,exec,/nix/store/q8flwbk5f20khbfrwbxdhr85n9zdrpnm-firefox-129.0/bin/firefox
bind=$mod,E,exec,/nix/store/6kw0higd4yd957rhmv73smxpwgh2vn8p-thunar-4.18.10/bin/thunar
bind=$mod,V,togglefloating,
bind=$mod,D,exec,pkill wofi || /nix/store/xmcw940x2bpynqh0yx80wkm2k6nk6sx7-wofi-1.4.1/bin/wofi -S drun
bind=$mod,P,pseudo,
bind=$mod,J,togglesplit,
bind=$mod,escape,exec,pkill wlogout || /nix/store/cpappxibmy5hh3s80ykdhjp2qx02x474-wlogout-1.2.2/bin/wlogout -p layer-shell
bind=$mod,F,fullscreen
bind=$mod,K,exec,/nix/store/3k5wiyk29z0x2d6w2dmi4la6yacc0j30-wofi-hyprkeys/bin/wofi-hyprkeys
bind=$mod, left, movefocus, l
bind=$mod, right, movefocus, r
bind=$mod, up, movefocus, u
bind=$mod, down, movefocus, d
bind=$mod CTRL, left, resizeactive, -5% 0
bind=$mod CTRL, right, resizeactive, 5% 0
bind=$mod CTRL, up, resizeactive, 0 -5%
bind=$mod CTRL, down, resizeactive, 0 5%
bind=$mod, mouse_down, workspace, e+1
bind=$mod, mouse_up, workspace, e-1
bind=$mod ALT, left, workspace, r-1
bind=$mod ALT, right, workspace, r+1
bind=$mod ALT, up, workspace, r+1
bind=$mod ALT, down, workspace, r-1
bind=$mod SHIFT, left, movewindow, l
bind=$mod SHIFT, right, movewindow, r
bind=$mod SHIFT, up, movewindow, u
bind=$mod SHIFT, down, movewindow, d
bind=,Print,exec,/nix/store/10325h9xnlcv1cz0z7k00gd1248m1ghf-grimblast-0.1/bin/grimblast --notify copy screen
bind=SHIFT,Print,exec,/nix/store/10325h9xnlcv1cz0z7k00gd1248m1ghf-grimblast-0.1/bin/grimblast --notify save screen
bind=CTRL,Print,exec,grimblast --notify copy area
bind=CTRL SHIFT,Print,exec,grimblast --notify save area
bind=$mod,1,workspace,1
bind=$mod,2,workspace,2
bind=$mod,3,workspace,3
bind=$mod,4,workspace,4
bind=$mod,5,workspace,5
bind=$mod,6,workspace,6
bind=$mod,7,workspace,7
bind=$mod,8,workspace,8
bind=$mod,9,workspace,9
bind=$mod SHIFT,1,movetoworkspacesilent,1
bind=$mod SHIFT,2,movetoworkspacesilent,2
bind=$mod SHIFT,3,movetoworkspacesilent,3
bind=$mod SHIFT,4,movetoworkspacesilent,4
bind=$mod SHIFT,5,movetoworkspacesilent,5
bind=$mod SHIFT,6,movetoworkspacesilent,6
bind=$mod SHIFT,7,movetoworkspacesilent,7
bind=$mod SHIFT,8,movetoworkspacesilent,8
bind=$mod SHIFT,9,movetoworkspacesilent,9
bindel=, XF86AudioRaiseVolume, exec, /nix/store/vhp7f25d0mj18gsin2m0ibjj1dj6697a-swayosd-0-unstable-2024-04-15/bin/swayosd-client --output-volume=raise
bindel=, XF86AudioLowerVolume, exec, /nix/store/vhp7f25d0mj18gsin2m0ibjj1dj6697a-swayosd-0-unstable-2024-04-15/bin/swayosd-client --output-volume=lower
bindel=, XF86MonBrightnessUp,exec, /nix/store/vhp7f25d0mj18gsin2m0ibjj1dj6697a-swayosd-0-unstable-2024-04-15/bin/swayosd-client --brightness=raise
bindel=, XF86MonBrightnessDown,exec, /nix/store/vhp7f25d0mj18gsin2m0ibjj1dj6697a-swayosd-0-unstable-2024-04-15/bin/swayosd-client --brightness=lower
bindl=, XF86AudioMute, exec, /nix/store/vhp7f25d0mj18gsin2m0ibjj1dj6697a-swayosd-0-unstable-2024-04-15/bin/swayosd-client --output-volume mute-toggle
bindl=, XF86AudioPrev, exec, /nix/store/qfqbcr7hn5j9hsn3sdbj6094i0p2vcvh-playerctl-2.4.1/bin/playerctl previous
bindl=, XF86AudioPlay, exec, /nix/store/qfqbcr7hn5j9hsn3sdbj6094i0p2vcvh-playerctl-2.4.1/bin/playerctl play-pause
bindl=, XF86AudioNext, exec, /nix/store/qfqbcr7hn5j9hsn3sdbj6094i0p2vcvh-playerctl-2.4.1/bin/playerctl next
bindm=$mod, mouse:272, movewindow
bindm=$mod, mouse:273, resizewindow
env=WLR_DRM_DEVICES,/home/sjcobb/.config/hypr/card
env=AQ_DRM_DEVICES,/home/sjcobb/.config/hypr/card
env=HYPLAND_TRACE,1
env=XDG_SESSION_TYPE,wayland
env=MOZ_ENABLE_WAYLAND,1
monitor=,highres,auto,auto

======Config-End========
```
</details>

### Description

@fufexan 

As of https://github.com/hyprwm/Hyprland/commit/df9d830117cbbf8a1b2a144a28261c28753cb022, the hyprpm Makefile was added that contains personal paths. I believe this was a mistake.

### How to reproduce

n/a

**Comment by vaxerski** [comment]
@fufexan idiot

**Comment by fufexan** [comment]
I blame cmake lol.

**Comment by fufexan** [comment]
Fixed with c5ec079c.
